src/main/modules/nunjucks/index.ts

```diff
@@ -118,6 +124,7 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('t', (key: string, options?: TranslationOptions): string => this.i18next.t(key, options))
     nunjucksEnv.addFilter('date', dateFilter)
     nunjucksEnv.addFilter('inputDate', dateInputFilter)
+    nunjucksEnv.addFilter('dateDay', dateDayFilter)
```

Comment:
`dateDay` does not look self explanatory in filter.

Reply:
renamed it to be something more descriptive.

Reply:
Thanks 👍